mud/lp-245/obj/master.c

```diff
@@ -712,16 +716,24 @@ object connect ()
 // binding the connection to it. That lfun has to return !=0 to succeed.
 
 {
-    object ob;
-    string ret;
```

Comment:
I think `ret` was dead-code and never set to a value in this old version. I changed to using `!ob` as an indicator to write a failure message to the user and matched that message to the "style" of the old code as best I could.